apiserver/internal/handlers/objects/objects.go

```diff
@@ -15,20 +15,24 @@ import (
 	jujuerrors "github.com/juju/errors"
 
 	apiservererrors "github.com/juju/juju/apiserver/errors"
-	coreapplication "github.com/juju/juju/core/application"
+	internalhttp "github.com/juju/juju/apiserver/internal/http"
 	"github.com/juju/juju/core/arch"
 	corecharm "github.com/juju/juju/core/charm"
-	coreunit "github.com/juju/juju/core/unit"
 	"github.com/juju/juju/domain/application"
 	"github.com/juju/juju/domain/application/architecture"
 	applicationcharm "github.com/juju/juju/domain/application/charm"
 	applicationerrors "github.com/juju/juju/domain/application/errors"
 	"github.com/juju/juju/internal/charm"
 	"github.com/juju/juju/internal/errors"
+	internallogger "github.com/juju/juju/internal/logger"
 	"github.com/juju/juju/rpc/params"
 	"github.com/juju/juju/state"
 )
 
+var (
+	logger = internallogger.GetLogger("juju.apiserver.objects")
```

Comment:
Maybe `juju.apiserver.handlers.objects`?

Reply:
We should pass in the logger, rather than depend on a global logger. I'll be happy to make that change in a subsequent PR.